x/wasm-storage/keeper/keeper.go

```diff
@@ -14,25 +14,13 @@ import (
 	"github.com/sedaprotocol/seda-chain/x/wasm-storage/types"
 )
 
-var (
-	// DataRequestPrefix defines prefix to store Data Request Wasm binaries.
-	DataRequestPrefix = collections.NewPrefix(0)
-	// OverlayPrefix defines prefix to store Overlay Wasm binaries.
-	OverlayPrefix = collections.NewPrefix(1)
-	// ProxyContractRegistryPrefix defines prefix to store address of
-	// Proxy Contract.
-	ProxyContractRegistryPrefix = collections.NewPrefix(2)
-	// ParamsPrefix defines prefix to store parameters of wasm-storage module.
-	ParamsPrefix = collections.NewPrefix(3)
-)
-
 // WasmKey takes a wasm as parameter and returns the key.
 func WasmKey(w types.Wasm) []byte {
```

Comment:
Same here... `GetWasmKey`

Reply:
in this commit: [9cd4dcd](https://github.com/sedaprotocol/seda-chain/pull/266/commits/9cd4dcdfb9578b193a3adce22a0095e8ab17c507)